src/Parser/Core/Modules/Items/DeceiversGrandDesign.js

```diff
@@ -0,0 +1,126 @@
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
+import Module from 'Parser/Core/Module';
+
+const debug = false;
+const GUIDING_HAND_DURATION = 120000;
+const PROC_EVENT_START_BUFFER = 5000;
+const PROC_EVENT_END_BUFFER = 1000;
+
+class DecieversGrandDesign extends Module {
+  healing = 0;
+  healingAbsorb = 0;
+  dgdTargetOne = null;
```

Comment:
Agree on the array. The one outstanding area here is tracking the HoT downtime and this is apart of what I initially was looking at to track the HoT downtime.

I am still working out a good way to implement that portion but I didn't want to hold up the rest of the module for a smaller enhancement.